pkg/controller/tailoredprofile/tailoredprofile_controller.go

```diff
@@ -191,85 +166,125 @@ func (r *ReconcileTailoredProfile) validateTailoredProfile(tp *compliancev1alpha
 	return true, nil
 }
 
-func (r *ReconcileTailoredProfile) getRulesFromSelections(tp *compliancev1alpha1.TailoredProfile) (map[string]*compliancev1alpha1.Rule, bool, error) {
-	rules := make(map[string]*compliancev1alpha1.Rule)
+// getProfileInfo gets the needed profile info (profile and profile bundle) from a tailored profile
+func (r *ReconcileTailoredProfile) getProfileInfo(tp *cmpv1alpha1.TailoredProfile) (*cmpv1alpha1.Profile, *cmpv1alpha1.ProfileBundle, error) {
```

Comment:
getProfileInfoFromExtends() has the same parameters, is this one needed?

Reply:
It isn't. at a previous implementation, I had gotten this to work with "profiles from scratch" but ended removing that. Let me clean this up.